packages/wallet/src/redux/protocols/virtual-funding/__tests__/reducer.test.ts

```diff
@@ -43,22 +43,29 @@ describe('happyPath', () => {
     itSendsTheseCommitments(result, [{ commitment: { turnNum: 0, allocation } }]);
   });
 
-  describe(scenarioStepDescription(scenario.openG), () => {
-    const { state, sharedData, action } = scenario.openG;
-    const { protocolState, sharedData: result } = reducer(state, sharedData, action);
+  describe.only(scenarioStepDescription(scenario.openJ), () => {
```

Comment:
What's the intended pattern with `only`? To use `only` during development and then remove all `only`'s when the tested reducer reaches a certain level of maturity?

Reply:
In this case, I had already defined scenarios/tests for an unimplemented reducer.

Perhaps I could have used `.skip` on those tests, rather than `.only` on whitelisted tests. All the tests are enabled in later PRs.